exporters/stats/prometheus/README.md

```diff
@@ -0,0 +1,58 @@
+# OpenCensus Prometheus Stats Exporter
+
+The *OpenCensus Prometheus Stats Exporter* is a stats exporter that exports data to 
+Prometheus. [Prometheus](https://prometheus.io/) is an open-source systems monitoring and alerting 
+toolkit originally built at [SoundCloud](https://soundcloud.com/).
+
+## Quickstart
+
+### Prerequisites
+
+To use this exporter, you need to install, configure and start Prometheus first. Follow the 
+instructions [here](https://prometheus.io/docs/introduction/first_steps/).
+
+### Hello "Prometheus Stats"
+
+#### Add the dependencies to your project
+
+TODO
```

Comment:
A kindly reminder: The readme files list in [RELEASING.md](https://github.com/census-instrumentation/opencensus-java/blob/master/RELEASING.md#update-release-versions-in-documentations-and-build-files) should be updated once this is released. :)

Reply:
Thanks, I've created https://github.com/census-instrumentation/opencensus-java/issues/966 to keep track of this.